package.json

```diff
@@ -46,5 +52,5 @@
     "feature flagging",
     "test track"
   ],
-  "license": "LICENSE"
+  "license": "MIT"
```

Comment:
Doesn’t matter much but LICENSE was meant to tell folks to read the LICENSE file. Maybe that’s not valid in yarn?

Reply:
Its not valid in NPM either, it produces a warning, same as in yarn.